lib/TestScriptRunnable.rb

```diff
@@ -56,9 +56,29 @@ def load_fixtures
 
       hash[fixture.id] = resource
       type = resource.resourceType
+      
+      # Need to consider where to put this code, such as a separate method before setup execition.
+      if fixture.autocreate
+        FHIR.logger.info "[.load_fixture] Autocreate Fixture: #{fixture.id}"
+        script.setup.action.each do |ac|
+            next if ac.operation == nil
+            if ac.operation.sourceId == fixture.id
+              FHIR.logger.warn "Possible duplication of fixture creation (autocreation and setup section)"
+              FHIR.logger.warn "Check setup in TestScript. Operation type: #{ac.operation.type.code} , sourceId: #{ac.operation.sourceId}"
+            end
+        end
```

Comment:
```suggestion
```
This code is unnecessary. While I appreciate the idea, there's no reason why a TestScript can't post a resource to a server using a fixture that has already been posted to the server. I can think of several cases where this might be the intended purpose, and not an accidental duplication.